docs/releases/i_0_0.md

```diff
@@ -0,0 +1,30 @@
+# iRobot® Create® 3 Release I.0.0
+[[Click here to download release I.0.0.CycloneDDS]](https://github.com/iRobotEducation/create3_docs/releases/download/I.0.0/Create3-I.0.0.CycloneDDS.swu)
+[[Click here to download release I.0.0.FastDDS]](https://github.com/iRobotEducation/create3_docs/releases/download/I.0.0/Create3-I.0.0.FastDDS.swu)
+
+!!! warning
+    Be sure to download the correct release for your choice of middleware.
```

Comment:
Do we want to call out that this release is differs from previous releases in this respect?  Maybe its clear enough from the 2 different clearly named files and this warning.

Reply:
I hoped the warning would be enough. I also call it out on the xml-config and application pages. If not, we can push another commit.